### ISSUE TITLE
/usr/lib32/libvulkan.so.1.3.264 from bundle wine is incompatible with Core™ 2nd generation

### ISSUE THREAD

**Issue by inferrna** [clearlinux/distribution#2965]
1. Go to the nearest museum for Core™ 2nd generation CPU
2. Clone vulkano project from https://github.com/vulkano-rs/vulkano/
3. `rustup toolchain install  stable-i686-unknown-linux-gnu`
4. In examples `cargo +stable-i686-unknown-linux-gnu build --bin teapot --release` (remove stupid from CFLAGS and CXXFLAGS if build fails)
5. Run `../target/release/teapot`
You will see:
```Rust
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: LibraryLoadFailure(DlOpen { desc: "/usr/lib32/libvulkan.so.1: CPU ISA level is lower than required" })', examples/src/bin/teapot/main.rs:68:40
```
Same time 64-bit teapot is working fine
md5sum:
`3cc5985f24009efc284eb60e1815e1a1  /usr/lib32/libvulkan.so.1.3.264`

Actually, I don't need this teapot, but also unable to run any 32-bit game from steam using proton. This teapot is just the most obvious way to see the actual reason of error.

**Comment by thiagomacieira** [comment]
Clear Linux does not support Core 2. The minimum processor supported are the original Core i3/i5/i7.

**Comment by inferrna** [comment]
Where did you see Core 2? I talk about Sandy/Ivy bridge. Personally, I have `Xeon E5-2680V2`.

![image](https://github.com/clearlinux/distribution/assets/4245277/69702cda-935e-4e8b-85c2-e50fd657f54a)
It is stupid Intel's CPU generation's naming, but you should be familiar with it.

**Comment by thiagomacieira** [comment]
Sorry,  my bad. You said "Core™ 2nd generation CPU" and I interpreted that as "Core 2", not "Second Generation Core-i3/i5/i7"

**Comment by inferrna** [comment]
Closed in favor of #2966
